modules/ROOT/pages/supported-cluster-config.adoc

```diff
@@ -69,22 +69,22 @@ You must configure a load balancer to use round-robin distribution of traffic am
 
 You must understand and adhere to the following recommendations and limitations when installing Anypoint Platform PCE and deploying apps.
 
-These limitations are based on a 6-node environment consisting of 6 nodes of 8 cores and 32GB RAM.
+These limitations are based on a 6-node PCE environment consisting of 6 nodes of 8 cores and 32GB RAM.
 
 **** Runtime management
 
-***** Maximum number of servers: 1000
+***** Maximum number of servers across all business groups: 1500 (or 750 Mule runtime clusters, consisting of 2 Mule runtime engines per cluster)
 ***** Maximum number of servers per server group: 20
 ***** Maximum servers per cluster: 8
 
-**** Applications per environment 
+**** Applications per business group environment 
 
-***** Mule applications under management:
+***** Mule applications under management per business groups environment:
 ****** Recommended: around 200
 ****** Maximum: 500
 
-***** Mule runtime engines under management
-****** Maximum: 1000 single Mule runtime engines (or 500 Mule runtime clusters, consisting of 2 Mule runtime engines per cluster)
+***** Mule runtime engines under management per business groups environment:
+****** Maximum: 500 single Mule runtime engines per business groups environment
 
 ***** Mule applications per Mule runtime engine
 ****** Maximum number of apps: 50 
```

Comment:
We should state the maximum number of Mule applications across all business group environments also.  Otherwise, it is left to the user to guess that.   For example, if maximum number of apps / server is 50 and you can have 1500 servers, does that mean you can have up to 75,000 applications?  The section below on API maximum created an implied Mule app maximum.  However, not all Mule App needs to implement an API.  So better to be explicit here.

Reply:
Yes, makes sense. Will add that.

Reply:
Done here https://github.com/mulesoft/docs-private-cloud/pull/168/commits/5ef33bab01f91b085b849c290db95bf8232436af